intercept.go

```diff
@@ -161,15 +162,16 @@ func InterceptMethod(intc InterceptorMethod, when When) {
 	})
 }
 
-func getInterceptors(when When, val reflect.Value) []*Interception {
-	result := []*Interception{}
+func getInterceptorEntries(when When, val reflect.Value) []*interceptionEntry {
+	result := []*interceptionEntry{}
```

Comment:
The controller (and `AppController`) value will be initialized for each request now, so that we should require large modifications to improve the further performance, I think.